config/admin_ranks.txt

```diff
@@ -11,7 +11,9 @@
 ##############################################################################################################
 # PLEASE NOTE: depending on config options, some abilities will be unavailable regardless if you have permission to use them!
 
-# KEYWORDS:
+# Follow the format below when documenting new keywords so the server tools may parse it
+
+# BEGIN_KEYWORDS
```

Comment:
json is not user editable because it does not support comments and it is too forward sensitive.

One typo can ruin the entire rest of the file parse wise with no ability to recover.

So it is not suitable for configs.

The way @Cyberboss is doing it is the proper way, doxy gen formats have existed for some time and it avoids duplicating information.